cpp/src/rolling/detail/range_window_bounds.hpp

```diff
@@ -78,35 +90,52 @@ using range_rep_type = typename range_type_impl<ColumnType>::rep_type;
 namespace {
```

Comment:
We are not supposed to have anonymous namespaces in header files.
https://github.com/rapidsai/cudf/blob/branch-22.10/cpp/docs/DEVELOPER_GUIDE.md#anonymous-namespace

Reply:
D'oh! Will fix.

Reply:
These are function/class templates. There isn't a need to protect against external linkage.

I'll just remove this unnamed namespace.

Reply:
Removed. Thanks for catching this.